src/test/java/org/llorllale/cactoos/matchers/InputHasContentTest.java

```diff
@@ -52,8 +56,29 @@ public void matchesInputContent() {
     public void failsIfContentDoesNotMatch() {
         new Assertion<>(
             "Matcher matches values that are not equal",
-            () -> new InputOf("hello"),
-            new IsNot<>(new InputHasContent("world"))
+            () -> {
+                final Description description = new StringDescription();
+                new InputHasContent("world").describeMismatchSafely(
+                    new InputOf("hello"), description
+                );
+                return description.toString();
+            },
+            new IsEqual<>("has content \"hello\"")
         ).affirm();
     }
+
+    @Test
+    public void describesExpectedValues() {
+        new Assertion<>(
+            "The matcher print the value which should be present in the "
```

Comment:
@borysfan Remove this string concatanation. Use `JoinedText` instead.

Reply:
I have changed it but IMHO it looks awkward. What is more I had to add exception to method signature.

Reply:
@borysfan You can use `new UncheckedText(new JoinedText(...))`.